gui/builtinViews/fittingView.py

```diff
@@ -488,12 +491,26 @@ def spawnMenu(self):
 
         while sel != -1 and sel not in self.blanks:
             mod = self.mods[self.GetItemData(sel)]
+
+            # Test if mod.isEmpty exists.
+            try:
+                mod.isEmpty
+            except AttributeError:
+                mod.isEmpty = False
```

Comment:
I'd rather have something like `if hasattr(mod, 'isEmpty') and not mod.isEmpty` rather than try catch here. Also, make a small note that simply says that t3d modes don't have this property, otherwise I'll be looking at this 2 years from now wondering why the fuck it's here lol